db/migrate/20160610124506_add_lives_on_to_middleware_server.rb

```diff
@@ -0,0 +1,11 @@
+class AddLivesOnToMiddlewareServer < ActiveRecord::Migration[5.0]
```

Comment:
there's another way to do it with the `change` method (and then there's no need for up/down) - I would leave that to miq core reviewers to define what is the preferred way